uportal-war/src/main/webapp/WEB-INF/flows/edit-portlet/choosePortlet.jsp

```diff
@@ -77,7 +77,7 @@ PORTLET DEVELOPMENT STANDARDS AND GUIDELINES
             </div>
             <!-- Application Dropdown -->
             <div class="form-group">
-                <div class=" col-md-3">
+                <div class=" col-md-3"> <spring:message code="application.information"/>
```

Comment:
Could you wrap the `<spring:message>` and the `<select>` in a `<label>`? That would improve the accessibility for this! 😄